index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <title>Fluidd</title>
+    <meta name="description" content="The Klipper web interface for managing your 3d printer" />
+
+    <link rel="icon" type="image/png" sizes="32x32" href="/img/icons/favicon-32x32.png" />
+    <link rel="icon" type="image/png" sizes="16x16" href="/img/icons/favicon-16x16.png" />
+    <meta name="theme-color" content="#2196F3" />
+    <meta name="mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <meta name="apple-mobile-web-app-status-bar-style" content="black" />
+    <meta name="apple-mobile-web-app-title" content="Fluidd" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/img/icons/apple-touch-icon-180x180.png" />
+    <link rel="mask-icon" href="img/icons/safari-pinned-tab.svg" color="#000000" />
+    <meta name="msapplication-TileImage" content="/img/icons/mstile-150x150.png" />
+    <meta name="msapplication-TileColor" content="#000000" />
+
+    <script type="module" src="/src/main.ts"></script>
+  </head>
+  <body>
+    <noscript>
+      <strong>We're sorry but Fluidd doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```

Comment:
Not sure if it makes sense to change this one, as we're using capitalized Fluidd in the readme too:
```suggestion
      <strong>We're sorry, but fluidd doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```